src/backend/utils/mmgr/mcxt.c

```diff
@@ -1526,3 +1588,7 @@ pgport_pfree(void *pointer)
 }
 
 #endif
+
+#ifdef EXTRA_DYNAMIC_MEMORY_DEBUG
+#include "../backend/utils/mmgr/mcxt_override.c"
```

Comment:
Please research how the same can be done without including .c file.

Reply:
I can not think of how it may be done another.